### PR TITLE
Expose feature and raster classification processes via wps.

### PR DESCRIPTION
**Pull request by jdeolive**
Adds a PPIO for the custom result returned by the recently added classification processes in GeoTools.  Along with a couple of unit tests. Also includes a modification to the states.properties file to create an actual distribution of data for testing.

### CODE REVIEW

File: src/extension/wps/wps-core/src/main/java/org/geoserver/wps/gs/ClassificationStatsPPIO.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
+/* (c) 2016 Open Source Geospatial Foundation - all rights reserved
+ * (c) 2001 - 2013 OpenPlans
```
Comment (jodygarnett) [inline src/extension/wps/wps-core/src/main/java/org/geoserver/wps/gs/ClassificationStatsPPIO.java]: new file 2001?

**Comment by jodygarnett** [comment]
You wanting this in for tomorrows code freeze @jdeolive ?

**Comment by jodygarnett** [comment]
merged to master

File: src/extension/wps/wps-core/src/main/java/org/geoserver/wps/gs/ClassificationStatsPPIO.java (hunk 1 above)
Comment (jdeolive) [inline src/extension/wps/wps-core/src/main/java/org/geoserver/wps/gs/ClassificationStatsPPIO.java]: I actually wrote the original 2-3 years back so kept the original copyright.

**Pull request closed without merging**